docs/UserGuide.md

```diff
@@ -474,31 +474,32 @@ If your changes to the data file makes its format invalid, JARVIS will discard a
-**Add studio** | `addstudio [l/LESSON_DESC] sd/START_DATE st/START_TIME [ed/END_DATE] et/END_TIME`                 | `addconsult l/Consultation on recursion sd/2022-10-14 st/12:00 et/14:00 si/3 si/4`
+Action | Format                                                                                           | Examples
+-------|--------------------------------------------------------------------------------------------------|----------
+**Help** | `help`                                                                                           |
+**List students** | `liststudent`                                                                                    |
+**List tasks** | `listtask`                                                                                       |
+**List lessons** | `listlesson`                                                                                     |
+**List all** | `listall`                                                                                        |
+**Clear** | `clear`                                                                                          |
+**Add student** | `addstudent s/NAME m/MATRIC_NUM`                                                                 | `addstudent s/John Doe m/A0123459G`
+**Delete student** | `deletestudent INDEX`                                                                            | `deletestudent 2`
+**Edit student** | `editstudent INDEX [s/NAME] [m/MATRIC_NUM]`                                                      | `editstudent 1 s/John Do`
+**Find student** | `findstudent KEYWORD [MORE_KEYWORDS]`                                                            | `findstudent John`
+**Add grade** | `grade INDEX [ra1/MARKS] [ra2/MARKS] [mt/MARKS] [pa/MARKS] [fn/MARKS]`                           | `grade 1 ra1/17 mt/55`
+**Set mastery check result** | `mc INDEX num/MC_NUM r/MC_RESULT`                                                                | `mc 1 num/1 r/PASS`
+**Add task** | `addtask t/TASK_DESC [d/DEADLINE]`                                                               | `addtask t/Prepare tutorial slides d/2022-11-01`
+**Delete task** | `deletetask INDEX`                                                                               | `deletetask 2`
+**Mark task** | `marktask INDEX`                                                                                 | `marktask 2`
+**Unmark task** | `unmarktask INDEX`                                                                               | `unmarktask 2`
+**Add consult** | `addconsult [l/LESSON_DESC] sd/START_DATE st/START_TIME [ed/END_DATE] et/END_TIME si/STUDENT_INDEX...` | `addconsult l/Consultation on recursion sd/2022-10-14 st/12:00 et/14:00 si/3 si/4`
+**Add studio** | `addstudio [l/LESSON_DESC] sd/START_DATE st/START_TIME [ed/END_DATE] et/END_TIME`                | `addstudio l/Studio 3 sd/2022-10-12 st/14:00 et/16:00`
```

Comment:
I think can swap the order for `addstudio` and `addconsult` for consistency in the ordering since I made the other commands follow the order used in the above part of the UG